external-providers/java-external-provider/Dockerfile

```diff
@@ -21,5 +21,5 @@ FROM quay.io/konveyor/jdtls-server-base
 COPY --from=builder /java-provider/java-external-provider /usr/local/bin/java-external-provider
 
 EXPOSE 14651
-
-ENTRYPOINT ["java-external-provider", "--port", "14651"]
\ No newline at end of file
+WORKDIR /addon
```

Comment:
Would `/tmp` be a better (more agnostic) default?

Reply:
@shawn-hurley @pranavgaikwad @fabianvf any opinions? `/tmp` sound good to me, but you all know far more about this project than I do.